suzieq/poller/inventory/inventory_sources/file.py

```diff
@@ -85,10 +91,45 @@ def _get_device_list(self) -> List[Dict]:
                         'devtype': devtype,
                         'namespace': nsname,
                         'ssh_keyfile': keyfile
-                    })
+                    }
+                    if self._validate_inventory_entry(entry):
+                        inventory.append(entry)
                 else:
                     logger.error(f'Ignoring invalid host spec.: {entry}')
 
         if not inventory:
             logger.error('No hosts detected in provided inventory file')
         return inventory
+
+    def _validate_inventory_entry(self, entry: dict) -> bool:
```

Comment:
```suggestion
    def _validate_inventory_entry(self, entry: Dict) -> bool:
```
It is only a matter of consistency with the notation used in the rest of the file